metric-anomaly-detector/build.gradle.kts

```diff
@@ -7,14 +7,37 @@ plugins {
 }
 
 application {
-    mainClass.set("org.hypertrace.alerting.metric.anomaly.detector.MetricAnomalyDetector")
+    mainClass.set("org.hypertrace.core.serviceframework.PlatformServiceLauncher")
+}
+
+tasks.test {
+    useJUnitPlatform()
 }
 
 dependencies {
     implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.23")
     implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.23")
 
+    implementation(project(":metric-anomaly-data-model"))
+
+    implementation("org.hypertrace.core.query.service:query-service-client:0.5.2")
+    implementation("org.hypertrace.core.attribute.service:attribute-service-client:0.9.3")
+    implementation("org.hypertrace.core.grpcutils:grpc-context-utils:0.3.4")
+    implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.23")
+    implementation("com.typesafe:config:1.4.1")
+    implementation("org.hypertrace.gateway.service:gateway-service-api:0.1.59")
```

Comment:
Is this use currently - `gateway-service-api` ?

Reply:
Can you also remove other unused libs?

Reply:
there are no unused libs